test/tests/ver-1kb/comparison_ver-1kb.py

```diff
@@ -0,0 +1,111 @@
+
+# Subplot 1: Time vs Pressure
+fig1 = plt.figure(figsize=[6, 5.5])
+ax1 = fig1.add_subplot(111)
+ax1.plot(TMAP8_time / 3600, TMAP8_pressure_enclosure_1, label=r"H2 Encl 1", c='tab:red', linestyle='dotted')
+ax1.plot(TMAP8_time / 3600, TMAP8_pressure_enclosure_2, label=r"H2 Encl 2", c='tab:blue', linestyle='dotted')
+ax1.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.1e}'.format(val)))
+ax1.set_xlabel('Time (hr)')
+ax1.set_ylabel('Pressure (Pa)')
+ax1.set_xlim(0, 3)
+ax1.set_ylim(bottom=0)
+ax1.legend(loc="best")
+ax1.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+fig1.savefig('ver-1kb_comparison_time.png', bbox_inches='tight')
+
+# Subplot 2: Pressure Encl 2 vs Concentration Encl 1 at interface
+k = np.sum(concentration_enclosure_1_at_interface * pressure_enclosure_2_at_interface) / np.sum(pressure_enclosure_2_at_interface ** 2)
+line_fit = k * pressure_enclosure_2_at_interface[1:]
+fig2 = plt.figure(figsize=[6, 5.5])
+ax2 = fig2.add_subplot(111)
+ax2.plot(pressure_enclosure_2_at_interface[1:], concentration_enclosure_1_at_interface[1:], marker='o', linestyle='None', c='tab:blue')
+ax2.plot(pressure_enclosure_2_at_interface[1:], line_fit, label=f'Fit: y = {k:.2e}x\n', c='tab:red', linestyle='--')
+ax2.set_xlabel(r"Pressure Encl 2 at interface (Pa)")
+ax2.set_ylabel(r"Concentration Encl 1 at interface (mol/m^3)")
+ax2.legend(loc="best")
+ax2.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+fig2.savefig('ver-1kb_comparison_concentration.png', bbox_inches='tight')
+
+# Subplot 3: Mass Conservation Sum Encl 1 and 2 vs Time
+fig3 = plt.figure(figsize=[6, 5.5])
+ax3 = fig3.add_subplot(111)
+ax3.plot(TMAP8_time / 3600, mass_conservation_sum_encl1_encl2, 'o', color='tab:blue', markersize=0.7)
+ax3.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.3e}'.format(val)))
+ax3.set_xlabel('Time (hr)')
+ax3.set_ylabel(r"Mass Conservation Sum Encl 1 and 2 (mol/m^3)")
+ax3.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+fig3.savefig('ver-1kb_mass_conservation.png', bbox_inches='tight')
+
+# Repeat the same for K=10/RT
+# Subplot 1 : Time vs Pressure
+fig1_k10 = plt.figure(figsize=[6, 5.5])
+ax1_k10 = fig1_k10.add_subplot(111)
+ax1_k10.plot(TMAP8_time_k10 / 3600, TMAP8_pressure_enclosure_1_k10, label=r"H2 Encl 1", c='tab:red', linestyle='-')
+ax1_k10.plot(TMAP8_time_k10 / 3600, TMAP8_pressure_enclosure_2_k10, label=r"H2 Encl 2", c='tab:blue', linestyle='-')
+ax1_k10.yaxis.set_major_formatter(plt.FuncFormatter(lambda val, pos: '{:.1e}'.format(val)))
+ax1_k10.set_xlabel('Time (hr)')
+ax1_k10.set_ylabel('Pressure (Pa)')
+ax1_k10.set_xlim(0, 3)
+ax1_k10.set_ylim(bottom=0)
+ax1_k10.legend(loc="best")
+ax1_k10.grid(which='major', color='0.65', linestyle='--', alpha=0.3)
+fig1_k10.savefig('ver-1kb_comparison_time_k10.png', bbox_inches='tight')
+
+# Subplot 2 : Pressure Encl 2 vs Concentration Encl 1 at interface
```

Comment:
Make sure to also calculate the difference between the expected final results and the calculated final results, and discuss how accurate TMAP8 is in the documentation.